groups/script.js

```diff
@@ -48,19 +56,26 @@ const groupsData = await getDiscordGroups();
 const groupRoles = document.querySelector('.groups-list');
 groupsData?.forEach((item) => {
   const group = document.createElement('li');
-  const groupname = document.createElement('p');
-  groupname.classList.add('group-name');
-  const createdBy = document.createElement('span');
-  createdBy.classList.add('create-by');
-  groupname.textContent = item.rolename;
-  createdBy.textContent = 'created by ' + membersIdNameObject[item.createdBy];
-  group.appendChild(groupname);
-  group.appendChild(createdBy);
   group.setAttribute('id', item.roleid);
   group.classList.add('group-role');
   if (window.location.search.slice(1) === item.rolename) {
     group.classList.add('active-group');
   }
+
+  const groupname = document.createElement('p');
+  groupname.classList.add('group-name');
+  groupname.setAttribute('id', `name-${item.roleid}`);
+  groupname.setAttribute('data-member-count', item.memberCount);
+  groupname.textContent = removeGroupKeywordFromDiscordRoleName(item.rolename);
+
+  const createdBy = createAuthorDetailsDOM(
+    item.firstName,
+    item.lastName,
+    item.image,
+  );
```

Comment:
since we are getting image URL from the backend API, suppose any user is there who do not have uploaded the image on the RDS server, then how are we going to handle that part, shouldn't we show the default image/avatar